ansitoimg/render.py

```diff
@@ -21,11 +21,11 @@
 TEXT_WIDTH = 12
 
 
-def _doRichRender(ansiText: str, wide: bool = True) -> Console:
-	console = Console(width=89 if wide else 49, record=True, file=StringIO())
+def _doRichRender(ansiText: str, wide: bool = True, width: int = 49) -> Console:
+	console = Console(width=round(width*1.82) if wide else width, record=True, file=StringIO())

```

Comment:
Would you be happy to pull this out into a separate variable? round(width*1.82) if wide else width